api/openapi-spec/swagger.json

```diff
@@ -11845,6 +11845,552 @@
       },
       "type": "object"
     },
+    "io.k8s.api.flowcontrol.v1alpha1.FlowDistinguisherMethod": {
```

Comment:
@liggitt any idea why updating the generated files causes this change?

Reply:
did you figure it out? this looks related to @deads2k change in https://github.com/kubernetes/kubernetes/pull/99362 ... did a `make clean` resolve it for you?

Reply:
I couldn't, I did ```make clean```, but didn't resolve the issue.

Reply:
I just reverted the irrelevant changes, but I am not sure if that is acceptable.

Reply:
what does `git describe` on your branch show?

Reply:
v1.21.0-alpha.1-3310-gb5ee0308a73

Reply:
that's why... your local tags are out of date... this alpha type is no longer served in 1.22

`git fetch --tags` should get you up to 1.22 and adjust the generator

Reply:
ah, thanks, now it works and is actually producing the same diff we have in this PR.